src/Containers/Microsoft.NET.Build.Containers/Tasks/CreateNewImage.cs

```diff
@@ -130,7 +130,7 @@ internal async Task<bool> ExecuteAsync(CancellationToken cancellationToken)
                             destinationImageReference,
                             message => SafeLog(message),
                             cancellationToken).ConfigureAwait(false);
-                        SafeLog("Pushed container '{0}' to registry '{1}'", destinationImageReference.RepositoryAndTag, OutputRegistry);
+                        SafeLog("Pushed image '{0}' to registry '{1}'", destinationImageReference, OutputRegistry);
```

Comment:
This PR is about including the image in this log message, so it can be easily copy-pasted, for example when you want to do a `docker run` next.